internal/commands/dates.go

```diff
@@ -152,16 +154,25 @@ func UpdateDatesByDialog(ctx context.Context, app *app.App, incidentDetails bot.
 		endDate            time.Time
 	)
 
+	logWriter := app.Logger.With(
+		log.NewValue("channelID", channelID),
+		log.NewValue("timeZoneString", timeZoneString),
+		log.NewValue("initDateText", initDateText),
+		log.NewValue("identificationDateText", identificationDateText),
+		log.NewValue("endDateText", endDateText),
+	)
+
+	logWriter.Debug(
+		ctx,
+		"command/close.CloseIncidentByDialog INFO",
```

Comment:
Is this right? The log says `CloseIncidentByDialog` while the name of the func is `UpdateDatesByDialog`. Same for the already existing (and maybe even repetitive, that should be removed) log in line 137.